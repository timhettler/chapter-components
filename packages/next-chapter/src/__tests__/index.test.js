import React from 'react';
import Component from '../';
import renderer from 'react-test-renderer';

it(`${Component.name} renders correctly`, () => {
  const tree = renderer
    .create(<Component title="FOO" subtitle="BAR" imgSrc="BUZ" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
