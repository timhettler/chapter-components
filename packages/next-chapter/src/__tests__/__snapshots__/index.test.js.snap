// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NextChapter renders correctly 1`] = `
<a
  className="nextChapter is-light show"
>
  <div
    className="copy"
  >
    <div
      className="title ui_next"
    />
    <div
      className="subtitle ui_next"
    />
  </div>
  <div
    className="thumbnail ui_next"
    style={
      Object {
        "backgroundImage": "url(undefined)",
      }
    }
  />
</a>
`;
